src/test/java/com/meilisearch/integration/classes/AbstractIT.java

```diff
@@ -69,6 +69,17 @@ public void loadResource(String fileName) throws IOException {
         testData.put(fileName, new TestData<>(rawData, movies));
     }
 
+    public Key getPrivateKey() throws Exception {
```

Comment:
Don't you need this kind of key in other tests? (just making sure that this is not a duplicated code somewhere)

Reply:
No, I just need it in the TokenTokenTest did you think it's better to add it only in `src/test/java/com/meilisearch/integration/TenantTokenTest.java`?

Reply:
Not really, could be useful in the future :)